internal/jobs/reddit.go

```diff
@@ -111,3 +111,16 @@ func processRedditResponse(j types.Job, resp []*reddit.Response, cursor client.C
 		NextCursor: cursor.String(),
 	}, nil
 }
+
+// GetStructuredCapabilities returns the structured capabilities supported by this Twitter scraper
+// based on the available credentials and API keys
+func (rs *RedditScraper) GetStructuredCapabilities() teetypes.WorkerCapabilities {
+	capabilities := make(teetypes.WorkerCapabilities)
+
+	// Add Apify-specific capabilities based on available API key
+	if rs.configuration.ApifyApiKey != "" {
```

Comment:
I think this is okay for now - but technically a miner can have an apify key which has NOT rented said actor.  Given our decentralized nature / incentive mechanism, I don't think this is an issue for the product (there will be miners who stay on top of what they need to rent), but perhaps we want to make a ticket to optimize detection of rented actors for a given provided key

Reply:
yes, I think there's a comment in the code somewhere but we should have a ticket for that. I'll add it.